src/lib/src/header/header.component.css

```diff
@@ -34,7 +34,7 @@ a.drawer {
   width: 20px;
   margin: 0 14px;
   line-height: 50px;
-  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAQAQMAAAAs1s1YAAAABlBMVEU2iqL///9rzgLQAAAAAnRSTlP/AOW3MEoAAAASSURBVAjXY2BAA///fwBjIsUBd4sLub+XcZUAAAAASUVORK5CYII=");
+  background-image: url("ngx-prx-styleguide/assets/images/ic_header_menu.png");
```

Comment:
I was sort of wondering if you could also do this by messing with the `src/demo/tsconfig.json` "paths" somehow.  Which could save you the trouble of replacing "ngx-prx-styleguide -> src/lib".

But I'm not sure that would actually work, and this already does